docs/sources/datahub-cloud-event-source.md

```diff
@@ -0,0 +1,114 @@
+can be granted using an [Access Policy](https://datahubproject.io/docs/authorization/access-policies-guide/). 
+
+## Overview
+
+The DataHub Cloud Event Source allows you to use DataHub Actions with an instance of DataHub Cloud hosted by [Acryl](https://acryl.io).
+
+Under the hood, the DataHub Cloud Event Source communicates with DataHub Cloud to extract change events in realtime.
+The state of progress is automatically saved to DataHub Cloud after messages are processed, allowing you to seamlessly pause and restart the consumer, using the provided `name` to uniquely identify the consumer state. 
+
+On initial startup of a new consumer id, the DataHub event source will automatically begin the _latest_ events by default. Afterwards, the message stream processed offsets will be continually saved. However, the source can also optionally be configured to "look back" in time 
+by a certain number of days on initial bootstrap using the `lookback_days` parameter. To reset all previously saved offsets for a consumer,
+you can set `reset_offsets` to `True`. 
+
+### Processing Guarantees
+
+This event source implements an "ack" function which is invoked if and only if an event is successfully processed
+by the Actions framework, meaning that the event made it through the Transformers and into the Action without
+any errors. Under the hood, the "ack" method synchronously commits DataHub Cloud Consumer Offsets on behalf of the Action. This means that by default, the framework provides *at-least once* processing semantics. That is, in the unusual case that a failure occurs when attempting to commit offsets back to Kafka, that event may be replayed on restart of the Action. 
+
+If you've configured your Action pipeline `failure_mode` to be `CONTINUE` (the default), then events which
+fail to be processed will simply be logged to a `failed_events.log` file for further investigation (dead letter queue). The DataHub Cloud Event Source will continue to make progress against the underlying topics and continue to commit offsets even in the case of failed messages. 
+
+If you've configured your Action pipeline `failure_mode` to be `THROW`, then events which fail to be processed result in an Action Pipeline error. This in turn terminates the pipeline before committing offsets back to DataHub Cloud. Thus the message will not be marked as "processed" by the Action consumer.
+
+## Supported Events
+
+The DataHub Cloud Event Source produces
+
+- [Entity Change Event V1](../events/entity-change-event.md)
+
+Note that the DataHub Cloud Event Source does _not_ yet support the full [Metadata Change Log V1](../events/metadata-change-log-event.md) event stream. 
+
+## Configure the Event Source
+
+Use the following config(s) to get started with the DataHub Cloud Event Source. 
+
+### Quickstart
+
+To start listening for new events from now, you can use the following recipe: 
+
+```yml
+name: "unique-pipeline-name"
+datahub:
+  server: "https://<your-organization>.acryl.io"
```

Comment:
```suggestion
  server: "https://<your-organization>.acryl.io/gms"
```

Reply:
i think the base URL also works. validated it does.

Reply:
i'd prefer simpler